atlas_core/helpers/classifications.py

```diff
@@ -0,0 +1,34 @@
+def classification_to_pandas(
+    df,
+    optional_fields=[
+        "name_es",
+        "name_short_en",
+        "name_short_es",
+        "description_en",
+        "description_es",
+        "is_trusted",
+        "in_rankings",
+        "reported_serv",
+        "reported_serv_recent",
+    ],
```

Comment:
Nothing to do here for now but I really don't like this growing list of extra fields we're allowing, and it feels like we'll need a better mechanism for this soon.